### ISSUE TITLE
Query about the use of dense annotation

### ISSUE THREAD

**Issue by butyuhao** [HKUST-KnowComp/VD-PCR#2]
Hello, from my understanding, in order to optimize retrieval metrics, VD-PCR is trained on the sparse annotation of the train set and evaluated on the sparse annotation of the val set. Then, to maximize the ranking metrics, VD-PCR is finetuned on the dense annotation of the train set and evaluated on the dense annotation of the val set. After that, the model is tested and the result was submitted to the test server. Is it correct?

**Comment by YuCosine** [comment]
Hi,

The training of VD-PCR on VisDial data contains two phases:

Phase 1: Sparse annotations are used. The model is trained on  the train set, evaluated on the val set, and then tested on the test set.

Phase 2: Dense annotations are used. The model is trained on  the train set and evaluated on the val set. Then, as explained in README, "for final inference on test set, use both train and val sets as training data to train the model." This is a common practice by previous VisDial models because dense annotations on the train set is too sparse.
